superdesk/io/ingest_provider_model.py

```diff
@@ -139,7 +140,7 @@ def _set_provider_status(self, doc, message=''):
     def on_create(self, docs):
         for doc in docs:
             if doc.get('content_expiry', 0) == 0:
-                doc['content_expiry'] = self.app.config['INGEST_EXPIRY_MINUTES']
+                doc['content_expiry'] = config.INGEST_EXPIRY_MINUTES
```

Comment:
So the key was not in app.config? If that is the case then I have to fix a lot more cases.